src/components/tree/reactD3Interface/change.js

```diff
@@ -47,6 +55,20 @@ export const changePhyloTreeViaPropsComparison = (mainTree, phylotree, oldProps,
   /* explode! */
   if (oldProps.explodeAttr !== newProps.explodeAttr) {
     args.changeNodeOrder = true;
+    args.newFocus = newProps.focus;
+  }
+
+  /* enable/disable focus */
+  if (oldProps.focus !== newProps.focus) {
+    args.newFocus = newProps.focus;
+    args.updateLayout = true;
+  }
+  /* re-focus on changes */
+  else if (oldProps.focus == true &&
+           newProps.focus == true &&
```

Comment:
Both these should use strict equality (`x === true`). In practice the supplied values are going to be `undefined`, `false` and `true`, so it shouldn't make a difference, but I'd like strict equalities used everywhere.

Reply:
https://github.com/nextstrain/auspice/pull/1872